### ISSUE TITLE
Purpose of *envrc-allow* and *envrc-deny* buffers

### ISSUE THREAD

**Issue by doolio** [purcell/envrc#50]
What is the purpose of these special buffers? They are just empty buffers for me or am I missing something. Any `direnv` related messages are relayed to me via the `*envrc*` buffer and/or the echo area. Couldn't the `*envrc*` buffer serve as the `DESTINATION` argument of `call-process`?

**Comment by purcell** [comment]
Yes, maybe! Needs some consideration, I think. I don't currently remember the original rationale. Probaby the idea was to display just the output of the single command that the user had explicitly run, but if the resulting buffer is empty (which is strange!) then that's not super helpful.

**Comment by purcell** [comment]
I just pushed a fix to remove those buffers when they're unnecessary.
